### DISCUSSION TITLE
How to find DBO Views in Azure Synapse

### DISCUSSION THREAD

**Discussion by TidyPyAi** [dbeaver/dbeaver#17672] [Q&A]
### Description

Dbeaver will not show the DBO Views or Tables
Connection = Active Directory MFA
Connection URL  =  Synapse Workspace Serverless URL;  from Azure Resource Group page.  
SSMS, Azure Data studio, SSDT, PowerBI: work just fine with this manner of connection

![image](https://user-images.githubusercontent.com/109158403/190213537-989dce8c-284d-4f05-8dab-41bb4ced03f3.png)

### DBeaver Version

22.2

### Operating System

windows 10

### Database and driver

Azure SQL 

![image](https://user-images.githubusercontent.com/109158403/190216223-5b4a018c-cb22-4eb5-9da6-0ceb447d5068.png)

### Steps to reproduce

Screenshot above.

### Additional context

NONE

**Comment by LonwoLonwo** [comment]
Hello @TidyPyAi

Unfortunately I can't reproduce your issue.

![2022-09-15 07_38_15-DBeaver Ultimate 22 2 0 - _MariaDB 10 1 3_Script-175](https://user-images.githubusercontent.com/45152336/190316045-0db73f0e-8f33-4de0-967a-0b446218f559.png)

Did you try to click on the "Schemas" folder in the Database Navigator tree and then choose "dbo" schema, click on dbo schema and then click on the "Views" folder?

**Comment by tidypy** [comment]
you are correct I did find it under schemas;  
It is in an unexpected place, but works for me. 

![image](https://user-images.githubusercontent.com/50213641/190431859-64368693-fa31-4bc2-996d-a9c35628995e.png)

**Comment by LonwoLonwo** [comment]
Hooray

Can I close this ticket?

Do you have any other questions?

**Comment by TidyPyAi** [comment]
Please close the ticket;  I did not see a place to close the ticket myself 😉

**Comment by LonwoLonwo** [comment]
Thanks for the answer
